clients/src/test/java/org/apache/kafka/common/requests/RequestResponseTest.java

```diff
@@ -113,7 +113,9 @@ public void testSerialization() throws Exception {
             checkSerialization(req, null);
 
         createMetadataResponse(0);
+        createMetadataResponse(1);
         createMetadataRequest(Arrays.asList("topic1")).getErrorResponse(0, new UnknownServerException());
+        createMetadataRequest(Arrays.asList("topic1")).getErrorResponse(1, new UnknownServerException());
```

Comment:
I just noticed that there's an existing bug here, we should pass these to `checkSerialization`.